llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -21228,6 +21228,14 @@ Value *RISCVTargetLowering::getIRStackGuard(IRBuilderBase &IRB) const {
   if (Subtarget.isTargetAndroid())
     return useTpOffset(IRB, -0x18);
 
+  Module *M = IRB.GetInsertBlock()->getParent()->getParent();
```

Comment:
I think `getParent()->getParent()` can be replaced with `getModule()`

Reply:
Thanks, will review.

Reply:
Yup, getModule is just `return getParent()->getParent();`. Done.